samples/sensor/mpr/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  name: MPR Pressure Sensor
+tests:
+  sample.sensor.mpr:
+    harness: sensor
+    tags: sensors
+    depends_on: i2c
+    filter: dt_compat_enabled("honeywell,mpr")
```

Comment:
we don't have any boards with this compatible, so this sample isn't going to be built in CI. i usually don't like whitelists, but in this case i think you should add arduino_due as a platform_whitelist instead of the filter.

Reply:
Ok, done.